src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -48,6 +73,67 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
   //    */
   // };
 
+  /**
+   * Creates and downloads an iCalendar file with the current activity's information.
+   */
+  const getICS = React.useCallback(() => {
+    const startTimeString = moment(props.startTime)
+      .tz(moment.tz.guess())
+      .format("YYYY-MM-DD-HH-mm");
+    const startTimeArr = startTimeString.split("-").map(Number);
+    const endTimeString = moment(props.endTime)
+      .tz(moment.tz.guess())
+      .format("YYYY-MM-DD-HH-mm");
+    const endTimeArr = endTimeString.split("-").map(Number);
+    const event = {
+      start: startTimeArr as ics.DateArray,
+      end: endTimeArr as ics.DateArray,
+      title: props.title,
+      description: `Attend Here: tamudatathon.com/events/activities/${props.id}`,
+      url: `https://tamudatathon.com/events/activities/${props.id}`,
+    };
+    ics.createEvent(event, (error, value) => {
+      if (error) {
+        console.log(error);
+        return;
+      }
+      const blob = new Blob([value], { type: "text/plain;charset=utf-8" });
+      FileSaver.saveAs(blob, "invite.ics");
+    });
+  }, []);
+
+  const remindMeMenu =
+    minsToEvent > 0 ? (
+      <>
+        <Dropdown>
```

Comment:
Can we make the dropdown use an outline style?
![image](https://user-images.githubusercontent.com/7832610/90057361-e6e98680-dca5-11ea-9a5f-d7aa88c19344.png)

Reply:
<img width="402" alt="Screen Shot 2020-08-12 at 3 23 32 PM" src="https://user-images.githubusercontent.com/34114190/90064020-c9b9b580-dcaf-11ea-87fa-a337521bc0fd.png">